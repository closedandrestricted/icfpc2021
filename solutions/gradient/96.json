{"vertices":[[36,151],[21,169],[18,140],[47,136],[44,163],[27,53],[31,40],[47,107],[60,82],[33,126],[58,116],[35,51],[56,71],[43,61],[55,105],[36,99],[66,100],[51,30],[48,90],[37,88],[49,122],[66,92],[63,116],[68,95],[26,88],[56,55],[72,91],[58,112],[59,41],[63,109],[65,105],[77,97],[79,93],[72,100],[67,115],[73,56],[75,106],[78,91],[78,111],[68,52],[89,89],[81,116],[76,98],[73,116],[95,92],[94,71],[95,101],[87,118],[109,110],[90,52],[102,66],[83,46],[104,84],[115,62],[109,32],[98,46],[128,127],[89,142],[122,143],[87,46],[100,42],[129,55],[125,68],[116,157],[136,150]]}