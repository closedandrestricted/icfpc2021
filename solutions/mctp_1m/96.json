{"vertices":[[24,138],[37,124],[14,124],[13,125],[29,107],[6,154],[17,148],[27,126],[14,152],[7,139],[8,147],[9,154],[23,143],[6,139],[3,140],[13,127],[12,133],[10,130],[5,141],[10,132],[20,141],[13,126],[14,140],[14,132],[11,142],[9,153],[13,126],[13,133],[4,140],[8,135],[5,138],[14,135],[12,138],[13,140],[13,127],[12,169],[9,145],[12,142],[13,148],[18,134],[16,146],[13,154],[12,150],[6,150],[13,140],[3,138],[20,134],[12,147],[7,143],[9,152],[5,147],[20,150],[12,127],[19,145],[9,157],[4,144],[12,168],[31,143],[18,181],[7,134],[5,147],[11,155],[23,150],[12,167],[4,149]]}