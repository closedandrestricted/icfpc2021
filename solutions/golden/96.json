{"vertices":[[34,147],[19,165],[16,136],[48,135],[42,163],[25,50],[31,40],[35,70],[60,85],[36,131],[57,117],[30,51],[51,74],[41,58],[57,108],[20,77],[65,101],[51,30],[48,93],[37,92],[51,124],[66,93],[56,109],[89,94],[29,87],[55,56],[85,99],[59,114],[57,42],[63,110],[65,106],[77,98],[79,94],[74,103],[67,114],[72,59],[73,110],[84,101],[73,116],[68,52],[89,89],[79,117],[84,94],[78,110],[95,92],[95,71],[95,101],[87,118],[109,110],[89,53],[103,66],[84,48],[105,84],[116,61],[108,33],[97,47],[128,127],[88,142],[122,143],[105,55],[101,42],[129,55],[126,68],[115,157],[135,150]]}